lib/ABI/Analyses/ConvertFunctionsToCABI.cpp

```diff
@@ -249,6 +249,11 @@ class ConvertFunctionsToCABI {
     auto ToConvert = filterTypes<RawFD>(Model->TypeDefinitions(),
                                         TypesToIgnore);
 
+    using NH = model::NamingHelper;
+    auto NamingHelper = Log.isEnabled() ?
+                          std::make_optional<const NH>(Model->namingHelper()) :
+                          std::nullopt;
```

Comment:
Use an `if` statement

Reply:
Ah, wait, this cannot be done as an `if` statement because the naming helper is not assignable (it has a reference member). Should I try to think of another trick? Or can a ternary operator stay?